security/models.py

```diff
@@ -162,7 +162,7 @@ class LoggedRequest(SmartModel):
     host = models.CharField(_('host'), max_length=255, null=False, blank=False, db_index=True)
     host._filter = CaseSensitiveStringFieldFilter
     method = models.SlugField(_('method'), max_length=7, null=False, blank=False, db_index=True)
-    path = models.CharField(_('URL path'), max_length=255, null=False, blank=True, db_index=True)
+    path = models.CharField(_('URL path'), max_length=2000, null=False, blank=True, db_index=True)
```

Comment:
A valid URL can be more than 255 chars long. I used 2000 which should be enough and we're using this value across projects for URL lengths.